### PR TITLE
Add local build description to README.md

### PR DESCRIPTION
**Pull request by samuelhafsteins**
The build script was incorrect, this should fix it and add some documentation to it.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -3,7 +3,11 @@ mkfile_dir:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 OPENAPI_HOST ?= https://app.aftra.io
 
 build:
-	go build cli/aftra/main.go
+	echo "Make sure you have go/bin in your PATH"
+	go get .
+	go install github.com/deepmap/oapi-codegen/cmd/oapi-codegen@v1.15.0
+	go generate ./...
```
Comment (sindrig) [inline Makefile:9]: These should maybe be moved to a separate target, something like `init` or `prepare`? I don't think `make build` should install and generate stuff.

File: Makefile [hunk 2]
```diff
@@ -3,7 +3,11 @@ mkfile_dir:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
...
+	go build ./...
```
Comment (sindrig) [inline Makefile:10]: Reason for `go build ./...` vs `go build cli/aftra/main.go`?

**Review by sindrig — APPROVED** [review]
LGTM - comments are for your consideration

File: Makefile (hunk 2 above)
Comment (samuelhafsteins) [reply Makefile:10]: `go build cli/aftra/main.go` doesn't exist

**Pull request merged**